tests/spread/pack/complex/imagecraft.yaml

```diff
@@ -0,0 +1,54 @@
+name: lotz-of-partitions
+version: "0.1"
+base: bare
+build-base: ubuntu@24.04
+
+platforms:
+  amd64:
+
+parts:
+  kernel:
+    plugin: nil
+    stage-packages: [linux-image]
+    organize:
+      "*": (volume/party/efi)/
+  rootfs:
+    plugin: nil
+    stage-packages: [busybox-static]
+    organize:
+      "*": (volume/party/rootfs)/
+
+volumes:
+  party:
+    schema: gpt
+    structure:
+      - name: efi
+        type: C12A7328-F81F-11D2-BA4B-00A0C93EC93B
```

Comment:
It might be nice to have convenience synonyms for the GUIDs for EFI system partitions and Linux partitions. (Out of scope here, more a design suggestion to think about separately)